_posts/2022-03-28-improving-readability-of-alias-commands.adoc

```diff
@@ -0,0 +1,207 @@
+44:35:85:d1:36:46:db:0b:51:58:6a:ae:f5:4d:ad:d8:65:8a:e1:68:04:34:bf:1b:d2:f2:60:f5:4f:33:df:c5:a1:7a:de:8e:54:7b:e5:20:29:eb:e9:40:11:15:07:aa:0c:f7:ba:69:00:60:98:44:73:e0:92:7f:a3:42:32:0c:a9:91:d6:7d:06:04:54:5c:21:dd:1d:f6:f4:96:3a:65:a3:2
+1:30:1f:30:1d:06:03:55:1d:0e:04:16:04:14:9c:85:1e:20:7f:82:fb:34:09:b2:8e:ad:30:9b:5d:60:e9:0c:13:2c:30:0b:06:09:60:86:48:01:65:03:04:03:02:03:3f:00:30:3c:02:1c:56:d1:21:21:47:29:76:15:88:c2:7f:d8:2e:0c:82:d3:fb:40:39:d8:f9:9a:2a:f7:87:1f:5b:ab
+:02:1c:54:66:42:6e:55:08:d5:4e:85:e3:e7:66:9f:8b:d8:ad:11:5e:63:91:60:9d:c2:a7:7b:ed:59:ce",
+            "subject" => "CN=www.example.com",
+            "issuer" => "CN=www.example.com",
+            "not-before" => "2022-03-28T10:21:44.000+0200",
+            "not-after" => "2022-06-26T10:21:44.000+0200",
+            "serial-number" => "08:f6:f0:9b:2c:35:23:9d",
+            "signature-algorithm" => "SHA256withDSA",
+            "signature" => "30:3c:02:1c:56:d1:21:21:47:29:76:15:88:c2:7f:d8:2e:0c:82:d3:fb:40:39:d8:f9:9a:2a:f7:87:1f:5b:ab:02:1c:54:66:42:6e:55:08:d5:4e:85:e3:e7:66:9f:8b:d8:ad:11:5e:63:91:60:9d:c2:a7:7b:ed:59:ce",
+            "version" => "v3"
+        }]
+    }
+}
+----
+
+Naturally, reducing verbosity of output, will make this command more user friendly, for instance:
+
+[source]
+----
+[standalone@localhost:9990 key-store=serverKS] :read-alias(alias="server1", verbose=false)
+{
+    "outcome" => "success",
+    "result" => {
+        "alias" => "server1",
+        "entry-type" => "PrivateKeyEntry",
+        "creation-date" => "2022-03-28T10:21:44.568+0200",
+        "certificate-chain" => [{
+            "type" => "X.509",
+            "algorithm" => "DSA",
+            "format" => "X.509",
+            "sha-1-digest" => "c5:cb:aa:b8:f9:4b:b0:44:59:ed:76:cd:ca:03:7d:df:d5:f6:ef:41",
+            "sha-256-digest" => "de:ef:8e:51:07:9e:f7:3a:aa:59:c5:13:3f:33:4d:bd:1b:8e:8c:34:50:36:89:2a:53:96:eb:7a:f9:ba:9e:ca",
+            "subject" => "CN=www.example.com",
+            "issuer" => "CN=www.example.com",
+            "not-before" => "2022-03-28T10:21:44.000+0200",
+            "not-after" => "2022-06-26T10:21:44.000+0200",
+            "serial-number" => "08:f6:f0:9b:2c:35:23:9d",
+            "signature-algorithm" => "SHA256withDSA",
+            "signature" => "30:3c:02:1c:56:d1:21:21:47:29:76:15:88:c2:7f:d8:2e:0c:82:d3:fb:40:39:d8:f9:9a:2a:f7:87:1f:5b:ab:02:1c:54:66:42:6e:55:08:d5:4e:85:e3:e7:66:9f:8b:d8:ad:11:5e:63:91:60:9d:c2:a7:7b:ed:59:ce",
+            "version" => "v3"
+        }]
+    }
+}
+----
+
+==== Making 'read-aliases' even better
+
+As mentioned above - read-aliases command bugged me, its name just does not align with output. So, obvious idea was to to incorporate 'read-alias' improvement with extra step, so following command:
```

Comment:
Maybe this could be reworded just to emphasize that this is now possible, e.g.:

As mentioned above - the `read-aliases` command bugged me, its name just does not align with output. So, a new `recursive` option has now been added to this command to actually display information about the aliases and a `verbose` option has also been added: